aspnet/fundamentals/middleware.rst

```diff
@@ -119,10 +119,10 @@ ASP.NET ships with the following middleware components:
 
 .. list-table:: Middleware
 	:header-rows: 1
-	
+
 	*  - Middleware
 	   - Description
-	*  - :doc:`Authentication </security/sociallogins>`
```

Comment:
I think this is going up one two many directories. Just make the link absolute: `/security/authentication/sociallogins`